docs/wiki/[User-Guide]-Starter-Module-Complete.md

```diff
@@ -0,0 +1,175 @@
+<!-- markdownlint-disable first-line-h1 -->
+The `complete` starer module provides full customization of the Azure Landing Zone through the means of a `config.yaml` file that provides the ability to enable and disable modules, configure module inputs and outputs, and configure module resources.
+A custom `config.yaml` file can be passed to the `additional_files` argument of the ALZ PowerShell Module. Allowing to firstly design your Azure Landing Zone, and then deploy it.
+If not specified, the default `config.yaml` file will be used, which is as follows:
+  
+  ```yaml
+---
+archetypes: # `caf-enterprise-scale` module, add inputs as listed on the module registry where necessary.
+  root_name: es
+  root_id: Enterprise-Scale
+  deploy_corp_landing_zones: true
+  deploy_online_landing_zones: true
+  default_location: uksouth
+  disable_telemetry: true
+  deploy_management_resources: true
+  configure_management_resources:
+    location: uksouth
+    settings:
+      security_center:
+        config:
+          email_security_contact: "security_contact@replace_me"
+    advanced:
+      asc_export_resource_group_name: rg-asc-export
+      custom_settings_by_resource_type:
+        azurerm_resource_group:
+          management:
+            name: rg-management
+        azurerm_log_analytics_workspace:
+          management:
+            name: log-management
+        azurerm_automation_account:
+          management:
+            name: aa-management
+connectivity:
+  hubnetworking: # `hubnetworking` module, add inputs as listed on the module registry where necessary.
+    hub_virtual_networks:
+      primary:
+        name: vnet-hub
+        resource_group_name: rg-connectivity
+        location: uksouth
+        address_space:
+          - 10.0.0.0/16
+        firewall:
+          name: fw-hub
+          sku_name: AZFW_VNet
+          sku_tier: Standard
+          subnet_address_prefix: 10.0.1.0/24
+        virtual_network_gateway: # `vnet-gateway` module, add inputs as listed on the module registry where necessary.
+          name: vgw-hub
+          sku: VpnGw1
+          type: Vpn
+          subnet_address_prefix: 10.0.2.0/24
+  ```
+
+> **Note:** We recommend that you use the `caf-enterprise-scale` module for management groups and policies, and the `hubnetworking` module for connectivity resources. However, connectivity resources can be deployed using the `caf-enterprise-scale` module if you desire.
+
+The schema for the `config.yaml` is documented here - [YAML Schema for `config.yaml`][wiki_yaml_schema_reference].
+
+## High Level Design
+
+![Alt text](./media/starter-module-hubnetworking.png)
+
+## Terraform Modules
+
+### `caf-enterprise-scale`
+
+The `caf-enterprise-scale` has been used to deploy the management group hierarchy, policy assignments and management resources. For more information on the module itself see [here](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale).
```

Comment:
Question. Why aren't we using the separate management resources module anymore?

Reply:
I think that module could be better for VNEXT, we'd need to modify the policies taking LAW as a parameter to take the custom name inputted in the separate ALZ Management module, using archetype overrides or otherwise. It could look a bit ugly.

Reply:
Sorry, I think I am getting confused with the example for the vNext. Matt used it in his, so I used it in complete_vnext.